docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
+
+2. Verify that the replication is up to date
+
+```
+$ commcare-cloud <env> run-shell-command pg1,pg2 'ps -ef | grep -E "sender|receiver"'
+
+    [ pg1 ] ps -ef | grep -E "sender|receiver"
+    postgres 5295 4517 0 Jul24 ? 00:00:01 postgres: wal sender process rep 10.116.175.107(49770) streaming 0/205B598
+
+    [ pg2 ] ps -ef | grep -E "sender|receiver"
+    postgres 3821 3808 0 Jul24 ? 00:01:27 postgres: wal receiver process streaming 0/205B598
+```
+
+Output shows that master and standby are up to date (both processing the same log).
+
+3. Promote the standby
+
 ```
-$ cchq <env> ansible-paybook promote_pg_standby.yml -e standy=[standby nde]
+$ commcare-cloud <env> ansible-paybook promote_pg_standby.yml -e standy=[standby node]
```

Comment:
should this say `standby`? Also it seems we have been using `<>` to show places where people should add their own values instead of `[]`.

Reply:
Yeah. I had copied that from another shard migration doc. Fixed it to `pg_standby` https://github.com/dimagi/commcare-cloud/pull/3072/commits/2292bf75e85799502b9fc1cadd696e254d609722